src/sagas/notifications.saga.js

```diff
@@ -1,12 +1,18 @@
 import { call, put, takeEvery } from 'redux-saga/effects';
 import { getNotifications } from '../api/notificationsApi';
-import { fetchNotificationsSuccess } from '../actions/notifications.action';
+import { fetchNotificationsSuccess,
+    fetchNotificationsFailed } from '../actions/notifications.action';
 
 function* fetchNotifications () {
-  const notifications = yield call(getNotifications);
+  try {
+    const notifications = yield call(getNotifications);
 
-  yield put(fetchNotificationsSuccess(notifications));
+    yield put(fetchNotificationsSuccess(notifications));
```

Comment:
error handling in notifications is assigned to our team,
let's stick to making changes specifically assigned to the team